bindata/network/multus-admission-controller/admission-controller.yaml

```diff
@@ -68,6 +71,13 @@ spec:
                   matchLabels:
                     hypershift.openshift.io/hosted-control-plane: {{.AdmissionControllerNamespace}}
                 topologyKey: kubernetes.io/hostname
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchLabels:
+                app: multus-admission-controller
+                hypershift.openshift.io/hosted-control-plane: {{.AdmissionControllerNamespace}}
```

Comment:
Do we need this line? With line 45, I think `app: multus-admission-controller` is sufficient here.